### PR TITLE
[Connect Widget] Fix retrieval of connected providers

### PR DESCRIPTION
**Pull request by leite08**
Ticket: #203

### Dependencies

- Upstream: none
- Downstream: none

### Description

- Fix retrieval of connected providers, it was missing `/token` on cloud environments
- improve API setup so we don't need to worry about that ^
- improve error handling
- introduce error dialog

### Release Plan

- anytime

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
+      "localRoot": "${workspaceFolder}/api/app",
+      "remoteRoot": "/usr/src/app"
+    },
+    {
+      "name": "Run Chrome",
+      "type": "chrome",
+      "request": "launch",
+      "url": "http://localhost:3001",
+      "webRoot": "${workspaceFolder}/connect-widget/app/"
+    }
```
Comment (leite08) [inline .vscode/launch.json:31]: Added this, the rest is just formatting

Comment (leite08) [reply .vscode/launch.json:31]: *the rest (in this file)...

File: README.md [hunk 2]
```diff
@@ -265,6 +265,8 @@ $ npm install # only needs to be run once
 $ npm run start # available on port 3001 by default
 ```
 
+To debug the Connect Widget, you can attach a run a Chrome window by launching the `Run Chrome` configuration in VS Code.
+
```
Comment (Goncharo) [inline README.md:269]: Awesome, I've been using the debugger in Chrome this whole time - this seems much better 🔥

**Review by Goncharo — APPROVED** [review]
LGTM! spaceSHIP IT 👽 🚀 👾

**Pull request merged**